src/lib/utils.js

```diff
@@ -68,7 +68,7 @@ let utils = {
       if (item[property] === undefined) {
         throw new Error(`[groupBy]: Object has no key ${property}`);
       }
-      let key = item[property];
+      let key = item[property] && item[property].toLowerCase();
```

Comment:
Hmmm, I think it's fine to group hits sharing the `null` value together. It happens if - for a unknown reason - you don't have `lvl1` for instance.

I can make the test more explicit if you want.